r2i/onnxrt/engine.cc

```diff
@@ -28,7 +28,7 @@ namespace onnxrt {
 Engine::Engine () {
   /* Initialize all variable members */
   logging_level = OrtLoggingLevel::ORT_LOGGING_LEVEL_WARNING;
-  intra_num_threads = 1;
+  intra_num_threads = 0;
```

Comment:
Since you found this value empirically, add a comment to explain why this is `0`